tests/README.md

```diff
@@ -0,0 +1,12 @@
+# Testing examples
+
+Examples are tested using tool [htrun](https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-host-tests) and templated print log. 
+
+To run the test, use following command after you build the example:
+```
+mbedhtrun -d D: -p COM4 -m K64F -f .\BUILD\K64F\GCC_ARM\error-handling.bin --compare-log tests\error-handling.log
```

Comment:
You could add that the command assumes that the target disk path (mount point) is D: and that serial port COM4 is used to communicate with K64F (DUT).

The binary is mbed-os-example-error-handling.bin not error-handling.bin.

If running on Mac, then forward slashes should be used, as well as a different DISK_PATH.

Reply:
This is fixed

Reply:
Shouldn't the path to the binary be .\BUILD\K64F\GCC_ARM\mbed-os-example-error-handling.bin?

Reply:
The name of the binary purely depend on the folder's name which the example been checked out to.  also different tool chain will have impact to the path of the binary. This command means to give user an example for user to follow, user might need to modify it if they need to.
I can change it to the default name, if that would make things easier.

Reply:
OK. Thanks for making the change anyway.